core/server/middleware/index.js

```diff
@@ -79,14 +78,17 @@ setupMiddleware = function setupMiddleware(blogApp) {
     // (X-Forwarded-Proto header will be checked, if present)
     blogApp.enable('trust proxy');
 
-    // Logging configuration
-    if (logging !== false) {
-        if (blogApp.get('env') !== 'development') {
-            blogApp.use(logger('combined', logging));
-        } else {
-            blogApp.use(logger('dev', logging));
-        }
-    }
+    /**
+     * request logging
+     * @TODO: add error to output
+     */
+    blogApp.use(function expressLogging(req, res, next) {
+        res.once('finish', function () {
+            logging.request({req: req, res: res});
+        });
```

Comment:
Have done some testing, we're not recording a duration. I think this needs improving, if not right now, in a later iteration.